libs/api/domains/auth/src/lib/loaders/client.loader.ts

```diff
@@ -0,0 +1,23 @@
+import { Injectable } from '@nestjs/common'
+import DataLoader from 'dataloader'
+
+import { NestDataLoader } from '@island.is/nest/dataloader'
+
+import { Client } from '../models/client.model'
+
+export type ClientDataLoader = DataLoader<string, Client>
+
+@Injectable()
+export class ClientLoader implements NestDataLoader<string, Client> {
+  async loadClients(clientIds: readonly string[]): Promise<Array<Client>> {
```

Comment:
Do we need a locale here as well?

Reply:
Yup, added a Input type. Wiring of this loader will then be done in a follow up PR.